sql/2021/performance/lighthouse_audits_corr_with_cwv.sql

```diff
@@ -0,0 +1,159 @@
+# standardSQL
+# Correlation between various lighthouse audits and core web vitals
```

Comment:
I would recommend rewriting this query similar to [this approach](https://github.com/HTTPArchive/almanac.httparchive.org/blob/4d10fdc6cf88b600e64f62a9f71b1de2a4842e9a/sql/2021/accessibility/lighthouse_a11y_audits.sql) used by the Accessibility team to analyze the entire a11y LH category (cc @obto)

The advantages being that it requires significantly less repetition in the SQL, doesn't need to be maintained in future years if audits are added/removed, and it transposes the results from columns to rows, which is a bit more ergonomic for working with the results.